Source/Flow/Public/Types/FlowEnumUtils.h

```diff
@@ -0,0 +1,126 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "Misc/EnumRange.h"
```

Comment:
these are some enum helper templates & macros that I use on our project, I finally broke-down and made a Flow version of them so I can do some static assets for the EFlowPinType